api/src/main/java/org/ccci/gto/globalreg/BaseGlobalRegistryClient.java

```diff
@@ -136,6 +151,18 @@ private void attachFilters(final Request request, final Filter... filters) {
         return this.serializer.deserializeEntity(type, response.content);
     }
 
+    private void addFieldsParameterIfNecessary(final Request request, final Set<String> fields) {
+        if (fields != null) {
+            request.queryParams.put(PARAM_FIELDS, COMMA_JOINER.join(fields));
+        }
+    }
+
+    private void addFullResponseParameterIfNecessary(final Request request) {
+        if (fullResponsesFromUpdates) {
+            request.queryParams.put("full_response", "true");
```

Comment:
I would add a constant for `full_response`

Reply:
good idea